pyproject.toml

```diff
@@ -41,6 +43,9 @@ black = {version = "^18.3-alpha.0", allow-prereleases = true}
 flake8 = "^3.7"
 Faker = "^4.0.0"
 
+[tool.poetry.plugins."fedora.messages"]
+"fas.group.member.sponsor.v1"="securitas.messaging.schema:MemberSponsorV1"
```

Comment:
I would recommend prefixing with `securitas` as it's not related to the topic but to the app.